packages/data-transformation/fhir-to-csv/src/parseFhir/configurations/config_MedicationAdministration.ini

```diff
@@ -21,8 +21,10 @@ identifier_2_value = identifier.2.value
 identifier_3_system = identifier.3.system
 identifier_3_type_text = identifier.3.type.text
 identifier_3_value = identifier.3.value
+identifier_4_system = identifier.4.system
+identifier_4_type_text = identifier.4.type.text
+identifier_4_value = identifier.4.value
 subject_reference = subject.reference
-subject_type = subject.type
```

Comment:
What's the plan for the cases where the subject is a group? Are we not supporting `Group`?